docs/docs/02-admins/03-operation/08-user-management.md

```diff
@@ -0,0 +1,111 @@
+---
+title: User Management
+---
+
+## Overview
+
+Renku uses [Keycloak](https://www.keycloak.org/) as its identity provider. When Renku is installed, the `init-keycloak-realms` Kubernetes job automatically initialises the Renku realm in Keycloak and sets up the necessary accounts, roles, and permissions.
+
+## Administrative Access
+
+### Accessing Keycloak Admin Console
+
+1. Navigate to your Keycloak instance (at `https://<your-domain>/auth`)
+2. Click on "Administration Console"
+3. Log in with your admin credentials
+4. Select the Renku realm to manage users and roles
+
+### Default Admin Credentials
+
+The default admin username and password are stored in the `keycloak-password-secret` Kubernetes secret when Renku is installed. To retrieve these credentials:
+
+```bash
+kubectl get secret keycloak-password-secret -n renku -o jsonpath='{.data.KEYCLOAK_ADMIN}' | base64 -d
+```
+
+```bash
+kubectl get secret keycloak-password-secret -n renku -o jsonpath='{.data.KEYCLOAK_ADMIN_PASSWORD}' | base64 -d
+```
+
+### Admin Role Assignment
+
+Users with the `renku-admin` role can perform administrative tasks in Renku, including:
+
+- Creating and managing resource pools
+- Configuring platform integrations
+
+## Managing Renku Admin Users
+
+To grant administrative privileges to additional users:
+
+1. Access the Keycloak Admin Console
+2. Navigate to the Renku realm
+3. Go to **Users** and select the target user
+4. In the **Role Mapping** tab, assign the `renku-admin` role
+
+## Managing Keycloak Admin Users
+
+In addition to Renku-specific admin user, you may want to create additional Keycloak admin accounts for managing the identity provider itself.
+
+### Creating Additional Keycloak Admins
```

Comment:
Do we want some more nuanced instructions here? For example, you can have just a user manager that can't also modify other parts of the configuration (clients, auth etc)